express-notes-api/index.js

```diff
@@ -0,0 +1,113 @@
+const express = require('express');
+const app = express();
+const fs = require('fs');
+const useJson = express.json();
+const jsonData = require('./data.json');
+let resArr = [];
+
+app.use(useJson);
+
+const pushJsonIntoArray = () => {
+  const eachNoteArr = Object.entries(jsonData.notes);
+  for (let i = 0; i < eachNoteArr.length; i++) {
+    resArr.push(eachNoteArr[i][1]);
+  }
+};
+
+app.get('/api/notes', (req, res) => {
+  resArr = [];
+  pushJsonIntoArray();
```

Comment:
If you're going to use this function, please make is return the array instead of updating a global variable.

Reply:
Done